flask_resty/spec/operation.py

```diff
@@ -0,0 +1,43 @@
+class Operation(dict):
+    """Exposes some utility methods to compose a swagger operation
+    https://github.com/swagger-api/swagger-spec/blob/master/versions/2.0.md#operation-object
+    """
+
+    def __init__(self):
+        super(Operation, self).__init__({
+            'responses': {},
+            'parameters': []
+        })
+
+    def add_parameter(self, location='query', **kwargs):
+        """Adds a new parameter to the request
+        :param location: the 'in' field of the parameter (e.g: 'query',
+            'body', 'path')
+        """
+        kwargs.setdefault('in', location)
+        if kwargs['in'] != 'body':
+            kwargs.setdefault('type', 'string')
+        self['parameters'].append(kwargs)
+        pass
+
+    def add_property_to_response(self, code='200', prop_name='data', **kwargs):
```

Comment:
swagger specs suggest to use strings instead of integers. Also to eliminate any possible difference arising from inserting 200 and "200"